### DISCUSSION TITLE
How to add Content-Encoding

### DISCUSSION THREAD

**Discussion by KevTale** [vercel/community#196] [Show & Tell]
Hi,

I'm using a unity webgl build, which display this message in the chrome console when I run it:

```
You can reduce startup time if you configure your web server to add "Content-Encoding: br" response header
```

Can someone tell me how to do this?

Thanks!

**Comment by Arihantar** [comment]
Have you tried disabling the compression format in Unity and see if that works?

**Comment by styfle** [comment]
Hi @KevTale 

`.unityweb` files are now compressed automatically, thanks!

If you need to configure additional headers, you can configure [`vercel.json`](https://vercel.com/docs/projects/project-configuration#headers).

[a comment by an automated account is not shown]
